packages/ui/src/ui/pages/navigation/Navigation/ContentViewer/helpers/checkContentIsSupported.ts

```diff
@@ -1,8 +1,35 @@
-import getComponentByMode from './getComponentByMode';
 import getComponentByContentType from './getComponentByContentType';
 import {Tab} from '../../../../../constants/navigation';
+import {YTError} from '../../../../../../@types/types';
+import ypath from '../../../../../common/thor/ypath';
+import getComponentByMode from './getComponentByMode';
 
-export default (type: string, mode: string) => {
+export function checkContentIsSupported(type: string, mode: string) {
     if (mode === Tab.CONTENT) return Boolean(getComponentByContentType(type));
     return Boolean(getComponentByMode(mode));
-};
+}
+
+function hasViewerForType(type: string): boolean {
+    return checkContentIsSupported(type, Tab.CONTENT);
+}
+
+export function itemNavigationAllowed(item: {
+    type: string;
+    targetPathBroken: unknown;
+    parsedPathError?: YTError;
+}) {
+    if (item) {
```

Comment:
`if(!item) return;`
So less nesting

Reply:
Just another moving of code, I haven't changed it.